### ISSUE TITLE
About labeled Regression Loss(Gaussian Probability)

### ISSUE THREAD

**Issue by JooYJang** [hikvision-research/ProbabilisticTeacher#7]
Hello, Thank you for sharing interesting research

I'm wondering why  gaussian pdf is directly used to formulate labeled regression loss  as a cross-entropy loss.
Is this generally used?
```
gaussian = gaussian_dist_pdf(fg_pred_deltas[..., :4],  gt_pred_deltas, sigma_xywh)
loss_box_reg_gaussian = - torch.log(gaussian + 1e-9).sum()
```
and gaussian_dist_pdf

```
def gaussian_dist_pdf(val, mean, var, eps=1e-9):
     simga_constant = 0.3
     return torch.exp(-(val - mean) ** 2.0 / (var + eps) / 2.0) / torch.sqrt(2.0 * np.pi * (var + simga_constant))
```
As far as I understand, the goal of mean and variance term used in this paper is to be gt_box (x_off, y_off, box_width, box_height) and zeros, since gt has direct delta fucntion with mean = gt_box and var = 0.
In this respect, directly using gaussian pdf as above would have infinite loss.

p.s/ How did you get sigma_constant = 0.3??

Thanks in advance
Joo Young Jang
